auth/services/selfsigned/types.go

```diff
@@ -32,6 +33,9 @@ const ContractFormat = "selfsigned"
 // VerifiablePresentationType is the dummy verifiable presentation type
 const VerifiablePresentationType = "NutsSelfSignedPresentation"
 
+// VerifiablePresentationTypeURI is the dummy verifiable presentation type as URI
```

Comment:
docstrings refer to dummy?

Reply:
Not sure, might be correct, might be not. Just copied from what's already there.